config/core/test-configs.yaml

```diff
@@ -2842,6 +2842,11 @@ test_configs:
   - device_type: sun50i-h5-libretech-all-h3-cc
     test_plans:
       - baseline
+      - baseline-nfs
```

Comment:
Passing in staging: https://staging.kernelci.org/test/plan/id/62a997d9ae114b994f74f1dc/